### ISSUE TITLE
librenms_msmtpd ignoring msmtpd.env?

### ISSUE THREAD

**Issue by jlixfeld** [librenms/docker#109]
I've modifed `msmtpd.env` but when trying to test an alert transport, it seems msmtpd isn't using the set values.  Specifically, it's still using the configuration in the container's /etc/msmtpd.conf

Am I missing a step somewhere?

```
(librenms) root@pgen:/opt/docker/librenms/librenms# more msmtpd.env
# https://github.com/crazy-max/docker-msmtpd
SMTP_HOST=204.11.50.150
SMTP_PORT=25
SMTP_TLS=off
SMTP_STARTTLS=off
SMTP_TLS_CHECKCERT=off
SMTP_AUTH=off
SMTP_USER=foo
SMTP_PASSWORD=bar
SMTP_FROM=msmtpd@msmtpd.lab.snickers.org
(librenms) root@pgen:/opt/docker/librenms/librenms# vi msmtpd.env
(librenms) root@pgen:/opt/docker/librenms/librenms# docker ps
CONTAINER ID        IMAGE                      COMMAND                  CREATED             STATUS                    PORTS                                                  NAMES
ff064927e4fd        librenms/librenms:latest   "/init"                  51 minutes ago      Up 51 minutes             514/tcp, 8000/tcp, 514/udp                             librenms_dispatcher
bc21e867f47e        librenms/librenms:latest   "/init"                  51 minutes ago      Up 51 minutes             0.0.0.0:514->514/tcp, 0.0.0.0:514->514/udp, 8000/tcp   librenms_syslogng
d6badf688265        librenms/librenms:latest   "/init"                  51 minutes ago      Up 51 minutes             514/tcp, 514/udp, 0.0.0.0:8000->8000/tcp               librenms
a95edbc2acce        mariadb:10.4               "docker-entrypoint.s…"   51 minutes ago      Up 51 minutes             3306/tcp                                               librenms_db
ea21836ceaaf        crazymax/rrdcached         "/entrypoint.sh /usr…"   51 minutes ago      Up 51 minutes (healthy)   42217/tcp                                              librenms_rrdcached
cfeca872837a        crazymax/msmtpd:latest     "/init"                  3 hours ago         Up 21 minutes (healthy)   2500/tcp                                               librenms_msmtpd
d484f4a387a8        memcached:alpine           "docker-entrypoint.s…"   3 hours ago         Up 51 minutes             11211/tcp                                              librenms_memcached
b2c6bef30097        redis:5.0-alpine           "docker-entrypoint.s…"   3 hours ago         Up 51 minutes             6379/tcp                                               librenms_redis
(librenms) root@pgen:/opt/docker/librenms/librenms# docker restart librenms_msmtpd
librenms_msmtpd
(librenms) root@pgen:/opt/docker/librenms/librenms# docker-compose logs -f
...
...
...
msmtpd_1      | msmtp: authentication failed (method PLAIN)
msmtpd_1      | msmtp: server message: 535-5.7.8 Username and Password not accepted. Learn more at
msmtpd_1      | msmtp: server message: 535 5.7.8  https://support.google.com/mail/?p=BadCredentials h18sm3204106iow.16 - gsmtp
msmtpd_1      | msmtp: could not send mail (account default from /etc/msmtprc)
msmtpd_1      | Aug 13 20:59:49 host=smtp.gmail.com tls=on auth=on user=foo from=LibreNMS@librenms.example.com recipients=me@here.com smtpstatus=535 smtpmsg='535-5.7.8 Username and Password not accepted. Learn more at\n535 5.7.8  https://support.google.com/mail/?p=BadCredentials h18sm3204106iow.16 - gsmtp' errormsg='authentication failed (method PLAIN)' exitcode=EX_NOPERM
```

```
(librenms) root@pgen:/opt/docker/librenms/librenms# docker exec -it librenms_msmtpd cat /etc/msmtprc
account default
logfile -
syslog off
host smtp.gmail.com
port 587
tls on
tls_starttls on
tls_certcheck on
auth on
user foo
password bar
from foo@gmail.com
(librenms) root@pgen:/opt/docker/librenms/librenms#
```

**Comment by crazy-max** [comment]
@jlixfeld As you can see in the [compose example](https://github.com/librenms/docker/blob/f21bdf7ef8ee2b2440a98b582b62753a29d95d03/examples/compose/docker-compose.yml#L58), `msmtpd.env` is only used by the `msmtpd` container not librenms. Also make sur you have set the SMTP server to `msmtpd` and SMTP port to `2500` in your Email Options on LibreNMS (/settings/alerting/email)

**Comment by crazy-max** [comment]
@jlixfeld Cannot reproduce your issue. `/etc/msmtprc` is populated for me with [our example](https://github.com/librenms/docker/tree/master/examples/compose). Please create a proper [bug report](https://github.com/librenms/docker/issues/new?template=bug_report.md) with all relevant info.

**Comment by crazy-max** [comment]
Closed in favor of #110
